django/thunderstore/community/api/experimental/views.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.community.api.experimental.serializers import (
+    CommunitySerializer,
+    PackageCategorySerializer,
+)
+from thunderstore.community.models import Community
+
+
+class CommunitiesExperimentalApiView(APIView):
+    def get(self, request, format=None):
+        communities = CommunitySerializer(Community.objects.listed(), many=True)
+        return Response(
+            {
+                "communities": communities.data,
+            },
+        )
+
+
+class PackageCategoriesExperimentalApiView(APIView):
```

Comment:
`ListAPIView` would work best here also, but requires a little more work because you need to resolve the community first.

Something like:
```python
class PackageCategoriesExperimentalApiView(ListAPIView):
    """
    Lists all package categories of a community
    """
    serializer_class = PackageCategorySerializer

    def get_queryset():
        # TODO: Replace `self.kwargs["community"]` for actual community reference
        return PackageCategory.objects.filter(community=self.kwargs["community"])
```

Reply:
Related: https://github.com/thunderstore-io/Thunderstore/pull/268#discussion_r597951654